### ISSUE TITLE
BUG: Not really a bug but a problem with the docs when plotting with cartopy

### ISSUE THREAD

**Issue by ocefpaf** [geopandas/geopandas#2652]
- [X] I have checked that this issue has not already been reported.

- [ ] I have confirmed this bug exists on the latest version of geopandas.

- [ ] (optional) I have confirmed this bug exists on the main branch of geopandas.

---

The documentation is probably mixing a bad cartopy/matplotlib version. You can either downgrade mpl or pin cartopy to >=0.21.

![Screenshot from 2022-11-22 11-58-39](https://user-images.githubusercontent.com/950575/203346804-61d60f4e-5980-4daf-85fc-83cdcfeaca5f.png)

I can send a PR if someone points me to how the docs are build.

**Comment by martinfleis** [comment]
That seems to be fixed on the latest version of the docs - https://geopandas.org/en/stable/gallery/cartopy_convert.html

**Comment by ocefpaf** [comment]
> That seems to be fixed on the latest version of the docs - https://geopandas.org/en/stable/gallery/cartopy_convert.html

Awesome. I was sure I tried the latest version but maybe I had a cache or something. Closing this.

**Comment by jorisvandenbossche** [comment]
I _think_ the latest versions work together, but you might not always get the latest versions depending on other dependencies in your environment (at least I ran into this last week: https://github.com/SciTools/cartopy/issues/2086#issuecomment-1256677646). But maybe that is solved now.

Should we still try to rebuild the stable docs (or add small patch) to get rid of the error?

**Comment by jorisvandenbossche** [comment]
I started with just triggering a new build on readthedocs: https://readthedocs.org/projects/geopandas/builds/18722662/

**Comment by jorisvandenbossche** [comment]
That looks good now -> https://geopandas.org/en/stable/gallery/cartopy_convert.html

**Comment by ocefpaf** [comment]
> I _think_ the latest versions work together,

I meant the latest version of the docs. Sorry I was not clear. Yes, the latest version of cartopy works well with mpl.

**Comment by jorisvandenbossche** [comment]
That was clear, it's probably my wording that was unclear ;)

It's still good you reported this and that we rebuilt the _stable_ docs to not have the error

**Comment by ocefpaf** [comment]
PS: I want to get rid of that proj "ERROR" message now but I'm failing to reproduce it locally. It doesn't affect the docs but make them a bit hard on the eyes.

**Comment by jorisvandenbossche** [comment]
Yeah, I noticed that error message as well, and I also can't reproduce it locally. It's strange, also on the readthedocs build, all the packages come from conda-forge.
